src/www/views/servers_view/server_card/index.ts

```diff
@@ -43,13 +43,19 @@ export class ServerCard extends LegacyElementMixin(PolymerElement) {
       }
       .card-header {
         display: flex;
+        align-items: flex-end;
       }
       .card-content {
         text-align: center;
         padding: 10% 0;
       }
       .card-header server-connection-indicator {
-        padding: 16px 0 0 16px;
+        width: 48px;
```

Comment:
These are just temporary values. I'll make the server card more flexible when I port it as well.